tests/python/relax/frontend/test_onnx_frontend.py

```diff
@@ -70,8 +70,39 @@ def generate_random_inputs(
 
 
 def check_correctness(
-    model: ModelProto, inputs: Optional[Dict[str, np.ndarray]] = None, opset: int = None
+    model: ModelProto,
+    inputs: Optional[Dict[str, np.ndarray]] = None,
+    opset: int = None,
+    legalize: bool = False,
+    target: str = "llvm",
 ) -> None:
+    @relax.transform.module_pass(opt_level=0)
+    def thread_bind(tvm_model: tvm.ir.IRModule, ctx: tvm.transform.PassContext):
```

Comment:
This is cool, would it be useful elsewhere? It seems like it might just be a better version of the ScheduleForTarget pass i tried to write.

Reply:
Hi Josh, this is a simplied version for auto-binding and written for the cases where MS cannot tune, i.e., dynamic shape. I think this could potentially be really useful pass to save some tuning but this might be not be very performance-friendly and may fail in complicated schedules. We can use it for testing for now and add it as a thread-binding only pass in the repo later :)

Reply:
@zxybazh would this pass generally support any operator? If so, I think it would be nice to promote it and apply it in the pass infra by default for GPU.

Reply:
Technically it should support any operator, just need more testing and see if there're cases where it doesn't work, e.g., blocks where they include other blocks. I agree it can be promoted to be applied by default, in case we don't want tuning.

Reply:
Got it. In that case, I suggest to add more tests and send this PR to the unity branch. This is super useful for other cases, especially for the testing and tutorials. Right now, whenever we use GPU, we should do tuning which seems pretty much overkill.

Reply:
Sounds good, would you please help me set up some test cases? That would help me figure out if there's any limitation in the current approach. Some othe previous tests where we have to do tuning would be great.

Reply:
I think testing ops in the following networks would be perfect. How about we create a list of ops used in those networks and split the work to set up the testcases? 
https://github.com/octoml/relax/blob/relax/python/tvm/meta_schedule/testing/relay_workload.py#L49